src/runtime/config/configuration-qemu.toml.in

```diff
@@ -33,6 +33,31 @@ machine_type = "@MACHINETYPE@"
 # Default false
 # confidential_guest = true
 
+# Enable pre-attestation AMD SEV and SEV-ES confidential guests.
+# Applies only if confidential_guest is true.
+# (default: false)
+#guest_pre_attestation = true
+#
+# Guest owner proxy that handles remote attestation
+#guest_pre_attestation_proxy="localhost:50051"
+#
+# Keyset ID for injected secrets
+#guest_pre_attestation_keyset="KEYSET-1"
+#
+# GUID of injected secret
+# Key Broker Server for SEV(-ES) expects secrets with this guid
+# See https://github.com/confidential-containers/simple-kbs
+#guest_pre_attestation_secret_guid="0a46e24d-478c-4eb1-8696-113eeec3aa99"
```

Comment:
Where does this hash come from?  Is this a harcoded one that would be always the same regardless of the deployment?  If not, in case it changes, what causes it to change and how would we update that?

Reply:
This value would change depending on the KeyBroker service in use (and the KeyBroker client baked into the guest agent). The user need to know the details of the attestation KeyBroker service in order to set the `secret_guid` and  `secret_type`.

Reply:
Let's say I have a fleet with 32 AMD machines.  All the machines would share the same guest_pre_attestation_secret_guid if they use the same KeyBroker service?

If another company has the a similar fleet, using their own instance of the KeyBroker service, would this be the same for both companies?

Reply:
This guid does not identify the guest. The guid identifies the secret that is injected into the guest. In other words, the KBC is expecting to find a secret with this guid. So this is basically set by the KBC and won't change as long as you are using the same KBC (regardlless of what KBS you use) It is configurable here because we just introduced the `online_sev_kbc` which expects to have a different secret identified by a different guid.

Reply:
Then I'd suggest having this one also as a Makefile variable that one can simply set during build time, when generating the payload that will be used with their fleet.

Reply:
Done